### PR TITLE
Wikidata to DBpedia class mapping generated from subClassOf

### PR DESCRIPTION
**Pull request by alismayilov**

### CODE REVIEW

File: core/src/main/scala/org/dbpedia/extraction/mappings/wikidata/WikidataR2RExtractor.scala [hunk 1]
```diff
@@ -56,9 +65,13 @@ class WikidataR2RExtractor(
                               DBpediaDatasets.WikidataSameAsExternal, DBpediaDatasets.WikidataNameSpaceSameAs, DBpediaDatasets.WikidataR2R_ontology)
 
 
+  private def loadConfigs(args:Array[String]): Unit ={
```
Comment (jimkont) [inline core/src/main/scala/org/dbpedia/extraction/mappings/wikidata/WikidataR2RExtractor.scala]: can you delete this?

File: core/src/main/scala/org/dbpedia/extraction/mappings/wikidata/WikidataR2RExtractor.scala [hunk 2]
```diff
@@ -219,18 +232,45 @@ class WikidataR2RExtractor(
...
+            }
+          })
+
       }
       case _ =>
     }
     classes
   }
 
+  private def readJson(fileName:String): Map[String,String] = {
```
Comment (jimkont) [inline core/src/main/scala/org/dbpedia/extraction/mappings/wikidata/WikidataR2RExtractor.scala]: generalize this function that appends the ontology mappings and only use the itemMap in the code
(better rename itemMap to classMappings)

Comment (jimkont) [inline core/src/main/scala/org/dbpedia/extraction/mappings/wikidata/WikidataR2RExtractor.scala]: and the class readClassMappings

File: core/src/main/scala/org/dbpedia/extraction/mappings/wikidata/WikidataR2RExtractor.scala (hunk 1 above)
Comment (alismayilov) [inline core/src/main/scala/org/dbpedia/extraction/mappings/wikidata/WikidataR2RExtractor.scala]: I have already deleted here: https://github.com/alismayilov/extraction-framework/commit/eea7203ba6775686f7da4b99113b893b4ec57015

Comment (jimkont) [inline core/src/main/scala/org/dbpedia/extraction/mappings/wikidata/WikidataR2RExtractor.scala]: maybe the branches are not correctly merged then

File: core/src/main/scala/org/dbpedia/extraction/mappings/wikidata/WikidataR2RExtractor.scala [hunk 3]
```diff
@@ -219,18 +230,49 @@ class WikidataR2RExtractor(
...
+                case _=>
+              }}
+            case _=>
+          }
+      }
+    } catch {
+      case ioe: IOException => println("Please check class mapping file "+ioe)
+    }
+
+    context.ontology.wikidataClassesMap.foreach {
```
Comment (jimkont) [inline core/src/main/scala/org/dbpedia/extraction/mappings/wikidata/WikidataR2RExtractor.scala]: we should check if the key exists before, if not, we keep as is, if it exists we combine the values.
This does not change any of the current results but if someone runs with newer version of the ontology and old mappings file we might get collisions

**Pull request merged**